ajax/libs/evaporate/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "evaporate",
+  "filename": "evaporate.min.js",
+  "version": "2.0.4",
+  "description": "Javascript library for browser to S3 multipart resumable uploads",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/TTLabs/EvaporateJS.git"
+  },
+  "keywords": [
+    "aws",
+    "amazon",
+    "S3",
+    "multipart",
+    "multi-part",
+    "uploads",
+    "browser",
+    "transfer",
+    "acceleration",
+    "signature version 4",
+    "v4"
+  ],
+  "author": {
+    "name": "Tom Saffell",
+    "email": "tcs@videopixie.com"
+  },
+  "license": "BSD 3-Clause",
```

Comment:
@x09326 we explicitly made the change previously to "BSD 2-Clause" because of compatibility with webjars user-submitted request). I kept the same format for that reason. But if webjars supports the short name you want, I will be happy to make the change to Evaporate...or apply a a solution that hopefully works for all platforms and distributions. Am open to suggestions.